src/data/Graph.ts

```diff
@@ -387,6 +387,51 @@ class GraphNode {
         }
         return dataIndices;
     }
+
+    getFullPathDataIndices(): {node: number[], edge: number[]} {
```

Comment:
Let's keep the naming style consistent. Please use `trajectory` instead of `fullPath`.